src/pages/AboutPage.js

```diff
@@ -39,68 +39,185 @@ const AboutPage = () => (
-          </a>{' '}to learn more.
+          The Tech and Design industries have had a long, racist, and non-inclusive history. To this day, these fields
+          perpetuate white supremacy and fail to properly represent Black folx. Software plays a huge role in our
+          society, but most tech is not created with Black folx in mind. In fact, lots of tech such as facial
+          recognition software that misidentifies Black people, not only fails to represent members of the Black
+          community, but in many cases has explicitly racist, dangerous, and even deadly effects.
         </p>
       </Col>
-      <Col xs={12} md={6}>
-        <CommPartnersIcon className="mx-auto d-block about-icon" fluid />
+      <Col xs={12} md={4}>
+        <TechDesign className="mx-auto d-block resource-icon" fluid />
       </Col>
     </Row>
 
-    <hr />
+    <hr className="resource-hr" />
 
-    <Row className="about-align-center">
-      <Col xs={12} md={6}>
-        <ReqProjectIcon className="mx-auto d-block about-request-icon" fluid />
+    <Row className="row-align-center">
+      <Col xs={12} md={4}>
+        <Logo className="mx-auto d-block resource-icon" fluid />
       </Col>
       <Col>
-        <h2>Request a Project</h2>
+        <h2>Where does Build for Black Lives fit in?</h2>
         <p>
-          Are you a Black business owner, activist, protest organizer, or community advocate looking for free services
-          building a website or online platform, creating social media content, data visualizations, or other projects?
+          This platform <strong>requires all project participants to be anti-racist</strong>, and be actively working to
+          dismantle their own biases and white-centric coding and design practices.
         </p>
-        <div className="about-button-wrapper">
-          <LinkContainer to="/project-request">
-            <Button className="primary-button" size="med">
-              Request a Project
-            </Button>
-          </LinkContainer>
-        </div>
+
+        <h4>We would like to make it clear that:</h4>
+        <ul>
+          <li>
+            <p>
+              Participating in this initiative is <strong>NOT</strong> an opportunity for participants to do "charity
+              work", cleanse their guilt, or impose their opinions about what the Black community needs.
+            </p>
+          </li>
+          <li>
+            <p>
+              Participating in this initiative <strong>IS</strong> an opportunity to create projects that are explicitly
+              requested by activists, organizers and Black business owners, and to provide free services, in order to
+              work towards justice and reparations for a community that has been systematically oppressed, exploited,
+              and underserved for hundreds of years.
+            </p>
+          </li>
+        </ul>
       </Col>
     </Row>
 
-    <hr />
+    <hr className="resource-hr" />
```

Comment:
could we still keep all this as a separate component and import? I think it'll be cleaner/easier to read since it's so long

Reply:
or alternatively turn all the links into a map would also work but not super important

Reply:
updated so that resources are now in their own json under 'data' like the FAQ, and then just mapped over in the About page